src/tx/builder/constants.js

```diff
@@ -0,0 +1,100 @@
+import BigNumber from 'bignumber.js'
```

Comment:
The schemas here are widely imported and used by the community developers. If we are moving them to a new file then we better add this to the migration guide. Super important!

Reply:
@davidyuk

Reply:
it is not a breaking change because I have reexported them as it was before:
https://github.com/aeternity/aepp-sdk-js/pull/1442/files#diff-ad38e160e81f081497c7c5146c2f092580b6e1fb9989415eae179c477dfbfb84R13
also
https://github.com/aeternity/aepp-sdk-js/pull/1442/files#diff-e496a95ddc06df5bbd37ef86211584337612c734f283228563dddb48e7b406deR33
https://github.com/aeternity/aepp-sdk-js/pull/1442/files#diff-119538feced9a8a6925bd1421ca798809cda6c6823bbff34af904a036f4b0d6bR33